docs/01-start-here/06-testing-tips.md

```diff
@@ -0,0 +1,157 @@
+# Testing tips
+
+### Get credentials from janus
+Before you do anything else on your local frontend build, navigate to the janus page
+and follow the steps to get your credentials for Frontend and for CMS-Fronts.
+This will save you grief later on.
+
+### Starting sbt
+Its important to remember to use ./sbt to run the local build, not just sbt.
+./sbt will pick up some presets that are important for running the build successfully.
+
+### What project to run
+In general, the best project to run is "dev-build".  Dev-build: is a project that combines multiple applications for ease of testing.
+Most of the time it is both convenient and reliable, however in some cases you might prefer to use the project containing the application you are changing.
+See: https://github.com/guardian/frontend/blob/master/dev-build/conf/routes - to learn more about what is supported in dev-build.
+
+To test a particular application. See the following for guidance on what content is served by what application:
+ https://github.com/guardian/frontend/blob/master/docs/02-architecture/01-applications-architecture.md
+
+### Running unit tests
+To run the scala unit tests for a given application, navigate to the project
+ that corresponds to the application in question and enter:
+```
+  test
+```
+To run all the scala unit tests. execute this command from the root project.
+
+To run only the tests for a particular test suite, use the command:
+```
+  test-only *<name-of-your-test-suite>
+```
+
+### How to test your devices on local
+- It is possible to use chrome emulator to get a good idea of how android devices behave and XCode the hardware simulators on XCode to do the same for iOS,
+However for significant frontend changes you should test on devices as much as possible. To do this you need to be able to make your localhost accessible.
+A tool called `ngrok` is a good way to make a local port accessible to external devices or computer (instructions on how to run ngrok [here](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/09-testing-externally-on-localhost.md))
+- To test on a variety of devices or to reproduce a bug on a particular set up, use https://www.browserstack.com. This site has a collection of computers and devices,
```

Comment:
Maybe instead -> 
`This tool contains both real and simulated versions of a most popular devices, browsers and operating systems. `
Not sure computers is the right terminology, OS would be more specific?